tests/utils.py

```diff
@@ -53,7 +52,7 @@ def assert_test_scene():
                 get_obj_cfg("bowl", "bowl", "ajzltc"),
                 get_obj_cfg("bagel", "bagel", "zlxkry", abilities=TEMP_RELATED_ABILITIES),
                 get_obj_cfg("cookable_dishtowel", "dishtowel", "Tag_Dishtowel_Basket_Weave_Red", prim_type=PrimType.CLOTH, abilities=TEMP_RELATED_ABILITIES),
-                get_obj_cfg("microwave", "microwave", "abzvij"),
+                get_obj_cfg("microwave", "microwave", "7128"),
```

Comment:
isn't this the outdated microwave asset?

Reply:
Yes, but this "abzvij" microwave causes issue for me because it has too many collision meshes. In any case, I will create a follow-up PR to get the tests working with the new asset.